### PR TITLE
Migrate database table changes for benchmarks

### PR DESCRIPTION
**Pull request by Digitalfear117**

### CODE REVIEW

**Review by Lekuruu — COMMENTED** [review]
This still won't work, because `DBBenchmark` doesn't have the hardware column.

**Comment by Digitalfear117** [comment]
Added

File: database/repositories/benchmarks.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ def create(
     score: int,
     grade: str,
     client: str,
+    hardware: str,
```
Comment (Lekuruu) [inline database/repositories/benchmarks.py:18]: This should probably be a `dict` now

**Review by Lekuruu — CHANGES_REQUESTED** [review]

File: database/repositories/benchmarks.py (hunk 1 above)
Comment (Digitalfear117) [reply database/repositories/benchmarks.py]: yes, I have updated it to dict

**Review by Lekuruu — APPROVED** [review]

**Pull request merged**